src/lib/api/org/index.ts

```diff
@@ -3,7 +3,7 @@ import { requestsManager } from 'snyk-request-manager';
 import * as debugLib from 'debug';
 import { getApiToken } from '../../get-api-token';
 import { getSnykHost } from '../../get-snyk-host';
-import { CreateOrgData, SnykProject } from '../../types';
```

Comment:
why are you dropping createOrgData here?

Reply:
that type is just not used in the file, it was redundant so I dropped it